dashboard/db/migrate/20170222192220_remove_href_from_external.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveHrefFromExternal < ActiveRecord::Migration[5.0]
+  def up
+    External.all.each{|level| level.update!(properties: level.properties.except('href'))}
+  end
```

Comment:
Does this make the migration irreversible?  If so, worth including a no-op `down` migration.